Mathlib/RingTheory/MvPowerSeries/PiTopology.lean

```diff
@@ -235,6 +242,46 @@ theorem instIsUniformAddGroup [AddGroup R] [IsUniformAddGroup R] :
 
 end Uniformity
 
+section ext
+
+variable {σ τ R S : Type*} [TopologicalSpace R] [TopologicalSpace S]
+
+lemma continuous_linearMap_ext [CommSemiring R] [AddCommMonoid S] [Module R S] [T2Space R]
+    [T2Space S] {f g : MvPowerSeries σ R →ₗ[R] S}
+    (hf : Continuous f) (hg : Continuous g) (h : ∀ i, f (monomial R i 1) = g (monomial R i 1))
+    (P : MvPowerSeries σ R) : f P = g P := by
+  rw [as_tsum P, Summable.map_tsum ⟨_, hasSum_of_monomials_self _⟩ _ hf,
+    Summable.map_tsum ⟨_, hasSum_of_monomials_self _⟩ _ hg]
+  congr 1
+  ext1 n
+  rw [← mul_one (coeff R n P), ← smul_eq_mul, map_smul, map_smul, h, map_smul]
+
+lemma continuous_ringHom_ext [Semiring R] [Semiring S] [T2Space R] [T2Space S]
```

Comment:
Idem, and there should be a `unique` theorem that shows that your `f` is `evalAlgHom` of something, and the same for `g`, hence there are the same.

Reply:
Actually I changed my mind and there is no point in keeping the theorems, it is more reasonable to go through polynomials assuming `CommRing` and maybe generalize later.